### PR TITLE
Cypress test for actions on rectangle.

### PR DESCRIPTION
**Pull request by dvkruchinin**
Cypress test for actions on rectangle.

Try to draw a rectangle shape in two ways (by 2 points, by 4 points)
Try to draw a rectangle track
Try to switch a label and draw one more rectangle
Check if objects are drawn and they are shapes/tracks with right labels

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: tests/cypress/support/commands.js [hunk 1]
```diff
@@ -88,28 +88,55 @@ Cypress.Commands.add('openTaskJob', (taskName, jobNumber=0) => {
     cy.openJob(jobNumber)
 })
 
-Cypress.Commands.add('createShape', (firstX, firstY, lastX, lastY) => {
+Cypress.Commands.add('createShape', (firstX, firstY, secondX, secondY, points='By 2 Points',
```
Comment (bsekachev) [inline tests/cypress/support/commands.js:91]: Can we sort these arguments? It's bad practice to mix argument with/without default value. I would suggest to pass here an object.

File: tests/cypress/support/commands.js [hunk 2]
```diff
@@ -88,28 +88,55 @@ Cypress.Commands.add('openTaskJob', (taskName, jobNumber=0) => {
...
+    if (points === 'By 4 Points') {
+        cy.get('.cvat-canvas-container')
+        .click(thirdX, thirdY)
+        cy.get('.cvat-canvas-container')
+        .click(fourthX, fourthY)
+    }
 })
 
-Cypress.Commands.add('createTrack', (firstX, firstY, lastX, lastY) => {
+Cypress.Commands.add('createTrack', (firstX, firstY, secondX, secondY, points='By 2 Points', thirdX, thirdY, fourthX, fourthY) => {
```
Comment (bsekachev) [inline tests/cypress/support/commands.js:121]: The same comment here

File: tests/cypress/support/commands.js [hunk 3]
```diff
@@ -88,28 +88,55 @@ Cypress.Commands.add('openTaskJob', (taskName, jobNumber=0) => {
...
+                                     thirdX, thirdY, fourthX, fourthY, switchLabel=false, labelName) => {
     cy.get('.cvat-draw-rectangle-control').click()
+    if (switchLabel) {
+        cy.get('.cvat-draw-shape-popover-content')
+        .find('.ant-select-selection-selected-value')
+        .click()
+        cy.get('.ant-select-dropdown-menu')
+        .contains(labelName)
+        .click()
+    }
```
Comment (bsekachev) [inline tests/cypress/support/commands.js:101]: Let's organize it as a different command to switch a label

**Review by bsekachev — CHANGES_REQUESTED** [review]

File: tests/cypress/support/commands.js (hunk 1 above)
Comment (dvkruchinin) [reply tests/cypress/support/commands.js]: Reworked to use the object as a parameter. Renamed the function for clarity. In this regard, I adapted tests that use this function.

File: tests/cypress/support/commands.js (hunk 2 above)
Comment (dvkruchinin) [reply tests/cypress/support/commands.js]: I deleted this command. Creating a rectangular track is created from a single command along with shape.

File: tests/cypress/support/commands.js (hunk 3 above)
Comment (dvkruchinin) [reply tests/cypress/support/commands.js]: Done.

**Review by bsekachev — APPROVED** [review]

**Pull request merged**